rebasehelper/helpers/lookaside_cache_helper.py

```diff
@@ -122,7 +122,8 @@ def download(cls, tool, basepath, package, target=None):
             cls._download_source(tool, url, package, source['filename'], source['hashtype'], source['hash'], target)
 
     @classmethod
-    def _upload_source(cls, url, package, filename, hashtype, hsh, auth=requests_gssapi.HTTPSPNEGOAuth()):
+    def _upload_source(cls, url, package, filename, hashtype, hsh, auth=requests_gssapi.HTTPSPNEGOAuth(),
+                       source_dir=''):
```

Comment:
I think it would make more sense if `source_dir` was positioned before `filename` argument.

Reply:
It definitely would but we would have to remove the default value as non-default arguments can't follow a default argument.

Reply:
Sure, is that a problem?